compute/rn/release-information/release-notes-22-01-update2.adoc

```diff
@@ -0,0 +1,105 @@
+* [SaaS] Fixes a navigation issue in the SaaS Console, where system admins, who have access to all Compute Console pages, are initially redirected to the *Radars > Containers* page with an "Invalid path" error when navigating to a page in Compute Console from a page outside Compute Console.
+
+// #36073
+* Fixes an issue in Cloud Discovery where the results returned in the Console UI were just HTTP 429 rate limit error messages.
+
+// #37034
+* Fixes an issue where a reverse shell attack wasn't properly identified on Alpine Linux.
+
+// #35222
+* Fixes an issue where Red Hat Enterprise Linux (RHEL) packages were incorrectly reported as vulnerable (for example, libxml2-python 2.9.1-6.el7_9.6).
+
+// #32760
+* Updates how the Prisma Cloud Compute scanning process impacts artifact metadata in JFrog Artifactory.
+The scanning process no longer updates the *Last Downloaded* date for all manifest files of all the images in the registry.
++
+NOTE: The *Last Downloaded* date of the manifest files of the images that are eventually pulled for scanning, based on your registry scan policy, will be updated.
+The scan process first evaluates which images to scan by retrieving all manifest files for all images.
+In this phase of the scan, the *Last Downloaded* date will no longer be impacted.
+In the next phase, where Prisma Cloud actually pulls an image to be scanned, the manifest file's *Last Downloaded* date will be updated.
+Often, the number of images scanned will be a subset of all images in the registry, but that's based on your scan policy.
++
+NOTE: Just because an image has been selected for scanning, doesn't mean that it will actually be pulled.
+If an image's hash hasn't changed, it won't be pulled for scanning, so the *Last Downloaded* date will be unchanged.
++
+NOTE: As part of the process for evaluating which images should be scanned, in addition to reviewing the manifest files, Prisma Cloud also examines the actual images.
+As part of examining the image files, the *Last Downloaded* date for these images files is updated.
+In the next release of Prisma Cloud, this will be fixed so that the *Last Downloaded* date won't change unless the image is actually pulled and scanned.
++
+This new capability is supported on JFrog Artifactory 7.21.3 and later.
+When configuring Prisma Cloud registry settings, version must be set to *JFrog Artifactory*.
+This capability isn't supported when version is set to  *Docker v2*.
+
+// #25822
+* Fixes an issue with runtime protection where audits and incidents were raised even when the events occurred in whitelisted file system paths.
+
+// #37065, #36960, #36798
+* Updates open source packages used in Prisma Cloud Compute.
+
+
+=== Known issues
+
+// #22837
+* When Defender is installed on Windows hosts in AWS, and Prisma Cloud Compute Cloud Discovery is configured to scan your environment for protected hosts, the Windows hosts running Defender are reported as unprotected.
+
+// PCSUP-8485 (on-prem only)
```

Comment:
@iansk , I believe this is fixed in update 2.

Reply:
Removed